packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -271,6 +300,17 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
     );
   }
 
+  private _renderSeparator(item: ICommandBarItemProps, index: number, isFarItem?: boolean): React.ReactNode {
+    const itemKey = item.key || String(index);
+    return (
+      <div
```

Comment:
We have a divider component that you can probably use here instead of taking a dependency  on ContextualMenuItem for this

Reply:
Done.